### PR TITLE
Right now

### LINKED ISSUE #42: "Right Now" component for Air Pressure

### LINKED ISSUE #43: "Right Now" component for Visibility

### LINKED ISSUE #44: "Right Now" component for Humidity

### LINKED ISSUE #45: "Right Now" component for Cloud Coverage

### LINKED ISSUE #46: "Right Now" component for Precipitation

### PR DESCRIPTION
**Pull request by tomasote**
Closes #46
Closes #45
Closes #44
Closes #43
Closes #42

### CODE REVIEW

File: front/app/components/ProfileMenu/LightModeSelectorComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <FontAwesomeIcon
+                icon={props.mode ? faMoon : faSun}
+                className="text-black text-2xl self-center self-align w-7 h-7"
+            />
+          </div>
+        </div>
+      </label>
+    </div>
+  );
+}
```
Comment (SimonDamberg) [inline front/app/components/ProfileMenu/LightModeSelectorComponent.tsx]: This component is not used, right?

File: front/utils/requestWrapper.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const getAPIEndpoint = () => {
     return "http://localhost:8000";
   } else {
     // If app is built in production, use prod environment
-    return "https://squid-app-znkgm.ondigitalocean.app";
+    return "http://localhost:8000";
```
Comment (SimonDamberg) [inline front/utils/requestWrapper.ts]: This should be reverted since deployed backend now works

**Review by SimonDamberg — CHANGES_REQUESTED** [review]
Looks good, nice work! Remove the unused Lightmode component and change back the URL for the API, then we can merge 👍🏼

**Comment by tomasote** [comment]
Requested changes fixed

**Pull request merged**